apm/server/methods/share.js

```diff
@@ -23,6 +23,16 @@ Meteor.methods({
     };
     var inviteId = PendingUsers.insert(inviteInfo);
     notifyPendingOwner(inviteId, app, email);
+
+    if (!Meteor.users.findOne({ username: email })) {
+     Accounts.createUser({
+       username: email,
+       email: email,
+       password: 'kadira2018',
```

Comment:
Would be great if you can set the password in settings file.

Reply:
OK, I'll make changes, soon.